.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: CI
```

Comment:
nit, but let's call this `tests.yaml` to be consistent with other PHP repos

Reply:
Renamed file to `.github/workflows/tests.yaml`